44334CFFFE342BB6.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "44334CFFFE342BB6",
+    "servers": [
+      {
+        "server_address": "router.eu.thethings.network",
+        "serv_port_up": 1700,
+        "serv_port_down": 1700,
+        "serv_enabled": true
+      }
+    ],
+    "ref_latitude": 46.85299921,
+    "ref_longitude": 7.58037771,
+    "ref_altitude": 0,
+    "contact_email": "",
```

Comment:
Could you please add an email address for contact? Thanks!

Reply:
Abdelaziz_o@hotmail.com, thanks for the pull..